superset/assets/package.json

```diff
@@ -67,6 +67,7 @@
     "bootstrap-slider": "^10.0.0",
     "brace": "^0.11.1",
     "classnames": "^2.2.5",
+    "core-js": "^2.6.0",
```

Comment:
Where is this one used?

Reply:
Without `core-js`, I get build failures like:
```
ERROR in ./node_modules/@superset-ui/chart/node_modules/@superset-ui/core/esm/models/Registry.js
Module not found: Error: Can't resolve 'core-js/modules/es6.array.iterator' in '/home/travis/build/mistercrunch/superset/superset/assets/node_modules/@superset-ui/chart/node_modules/@superset-ui/core/esm/models'
 @ ./node_modules/@superset-ui/chart/node_modules/@superset-ui/core/esm/models/Registry.js 1:125-168
 @ ./node_modules/@superset-ui/chart/node_modules/@superset-ui/core/esm/index.js
 @ ./node_modules/@superset-ui/chart/esm/registries/ChartTransformPropsRegistrySingleton.js
 @ ./node_modules/@superset-ui/chart/esm/index.js
 @ ./src/addSlice/AddSliceContainer.jsx
 @ ./src/addSlice/App.jsx
 @ ./src/addSlice/index.jsx
 @ multi babel-polyfill ./src/preamble.js ./src/addSlice/index.jsx
```

Reply:
`core-js` appears to be polyfill-related, I'm unclear if it's a build issue with `superset-ui`, some babel issue or something else, but adding `core-js` fixes it.

Reply:
It should be imported via `babel-polyfill`. Probably need to update `@superset-ui/core` config to list that as `dependency`.

Reply:
Fixed all issues by upgrading to babel 7.*